scripts/prepare_heat_data.py

```diff
@@ -32,13 +35,7 @@
 
 SHARE_DISTRICT_HEAT = 0.1
 
-CALIBRATE_LOAD = True
-
-# TODO Add to Snakemake
-CALIBR_DIR = snakemake.input.calib_dir
-
-CALIBR_HEAT_FL = "mod_heating_calibr_clean.csv"
-CALIBR_COOL_FL = "mod_cooling_calibr_clean.csv"
+CALIBRATE_LOAD = False
```

Comment:
It would be advisable to have few config options like:
space_heating_demand:
    mode: "default_with_scaling" # "default_with_scaling" applies the scaling proposed here, in the future we could have "EnergyPlus" or ResStockComStock to enable the loading of the E+ simulations for example
    total_heat_demand: ...
    
hard-coded values are too risky to be included in scripts

Reply:
Thank you! That is the adjustment which should be aligned with  the project-focused PR which introduces Energy+ outputs into `geothermal-modeling` repo (this one https://github.com/open-energy-transition/geothermal-modelling/pull/60)

Reply:
Definitely, the proposal was to trigger the E+ loading with that same option if we decide to go for that